test/e2e/cypress/plugins/index.js

```diff
@@ -55,5 +55,91 @@ module.exports = (on, config) => {
       });
       return null;
     },
+
+    selectChecks({ clusterId, checkIds }) {
```

Comment:
We already have a command for this:
https://github.com/trento-project/web/blob/main/test/e2e/cypress/support/commands.js#L64

I think commands are the way to go, but well, in any case, let's have it only in 1 place hehe